jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererAutoTriggerService.kt

```diff
@@ -171,36 +172,61 @@ class CodeWhispererAutoTriggerService : CodeWhispererAutoTriggerHandler, Disposa
 
         val rightContext = caretContext.rightFileContext
         val lengthOfRight = rightContext.trim().length
-        val triggerTypeCoefficient = CodeWhispererClassifierConstants.triggerTypeCoefficientMap[automatedTriggerType] ?: 0.0
+
+        val isExperimentGroup = CodeWhispererUserGroupSettings.getInstance().getUserGroup() == CodeWhispererUserGroup.Classifier
+
+        val triggerTypeCoefficient = if (isExperimentGroup) {
+            CodeWhispererClassifierConstants.triggerTypeCoefficientMapExp[automatedTriggerType] ?: 0.0
+        } else CodeWhispererClassifierConstants.triggerTypeCoefficientMap[automatedTriggerType] ?: 0.0
 
         val osCoefficient: Double = if (SystemInfo.isMac) {
-            CodeWhispererClassifierConstants.osMap["Mac OS X"] ?: 0.0
+            if (isExperimentGroup) {
+                CodeWhispererClassifierConstants.osMapExp["Mac OS X"] ?: 0.0
+            } else CodeWhispererClassifierConstants.osMap["Mac OS X"] ?: 0.0
         } else if (SystemInfo.isWindows) {
             val osVersion = SystemInfo.OS_VERSION
             if (osVersion.contains("11", true)) {
-                CodeWhispererClassifierConstants.osMap["Windows 10"]
+                if (isExperimentGroup) {
+                    CodeWhispererClassifierConstants.osMapExp["Windows 10"]
```

Comment:
just to confirm we are using the same "Windows 10" key for both windows 11 and windows 10? If yes do we want to combine these two?

Reply:
yes it does make sense to combine will do